src/kanata/mod.rs

```diff
@@ -38,6 +38,7 @@ pub struct Kanata {
     pub prev_layer: usize,
     pub scroll_state: Option<ScrollState>,
     pub hscroll_state: Option<ScrollState>,
+    pub move_mouse_state: Option<MoveMouseState>,
```

Comment:
I think it would be better to split this into vertical and horizontal states so that diagonal movements are supported.